roles/teuthology/tasks/apt_systems.yml

```diff
@@ -0,0 +1,16 @@
+---
+- name: Include package type specific vars.
+  include_vars: "{{ ansible_pkg_mgr }}_systems.yml"
+  tags:
+    - always
+
+- name: Install packages via apt
+  apt:
+    name: "{{ item }}"
+    state: latest
+    update_cache: yes
+    cache_valid_time: 600
+  no_log: true
```

Comment:
Why do we not want this logged? If for some reason we fail while installing packages that log might be useful.